lib/pdk/cli/validate.rb

```diff
@@ -17,6 +17,11 @@ module PDK::CLI
     flag :a, 'auto-correct', _('automatically correct problems (where possible)')
 
     run do |opts, args, _cmd|
+      if args == ['help']
```

Comment:
Is there a Cri way to do this? For other commands (test, new) it looks like we get the help subcommand for free. But it looks like 'validate' is different since on it's own (just 'pdk validate') it is a valid command